lib/shared/utils.js

```diff
@@ -48,22 +48,15 @@ function timecodeToMicroseconds(timecode, fps) {
 }
 
 function extractStyling(text) {
-  let unformattedText = '';
-  if (text.search(/<br>/) && !text.startsWith('<br>')) {
-    unformattedText = text.replace(/<br>/, '\n');
-  }
-  const formattingTags = unformattedText.match(/<[^>]*>|{[^}]*}/g);
-  if (formattingTags && formattingTags.length) {
-    formattingTags.forEach(tag => {
-      if (tag.search(/{[^}]*}/) > -1) {
-        unformattedText = unformattedText.replace(tag, ' ').replace(/\s\s+/, ' ');
-      } else {
-        unformattedText = unformattedText.replace(tag, '');
-      }
-    });
-  }
-
-  return unformattedText.trim();
+  const regexReplace = [
```

Comment:
It would be helpful to write a brief comment next to each line summarizing what is being done.